code/Model/SiteTree.php

```diff
@@ -137,6 +137,10 @@ class SiteTree extends DataObject implements PermissionProvider, i18nEntityProvi
      */
     protected static $_allowedChildren = array();
 
+    private static $show_stage_link = true;
+
+    private static $show_live_link = true;
```

Comment:
In this base class, could you add a comment that explains what these do in the practical sense? Something as simple as "set to false to disable the CMS preview for this class"

Reply:
Commented 👍